src/main/java/frc/robot/posecalcs/PoseManager.java

```diff
@@ -0,0 +1,12 @@
+package frc.robot.posecalcs;
+
+public class PoseManager {
+    private PoseCalcs poseCalcs;
```

Comment:
i think probably PoseManager will compute a new value given some inputs, rather than storing a pose.  The danger of storing a value is that the caller doesnt really know when the value was computed-- was it recently? 

I'd prefer a method like:

RobotPose  newPose = PoseManager.computePose ( RobotPose visionPose, RobotPose navxPose );

the idea with this signature is that we always combining input from the various sources.